distributed-tests/distributed-server.yml

```diff
@@ -0,0 +1,119 @@
+---
+- hosts: all
+  tasks:
+  - lineinfile:
+      dest: /etc/sysconfig/network-scripts/ifcfg-eth0
+      regexp: ^IPV6INIT=
+      line: IPV6INIT=no
+    name: Disable ipv6 in eth0 config
+
+  - lineinfile:
+      dest: /etc/sysconfig/network
+      regexp: ^NETWORKING_IPV6=
+      line: NETWORKING_IPV6=no
+    name: Disable ipv6 in network config
+
+  - sysctl:
+      name: "{{ item }}"
+      value: 1
+      state: present
+      reload: yes
+    with_items:
+    - net.ipv6.conf.all.disable_ipv6
+    - net.ipv6.conf.default.disable_ipv6
+    name: Disable ipv6 in sysctl
+    notify:
+    - run dracut
```

Comment:
So the way I originally planned it doesn't work anymore. Perhaps we should add a task here to run dracut and restart the server. See this [question](https://stackoverflow.com/questions/23877781/how-to-wait-for-server-restart-using-ansible) for tips on how to restart server and wait for it to come back online.